source/about/cloud-dedicated.rst

```diff
@@ -0,0 +1,114 @@
+
+Secure networking
+~~~~~~~~~~~~~~~~~~
+
+Mattermost Cloud Dedicated supports :doc:`IP filtering </manage/cloud-ip-filtering>` through CIDR-based IP ranges, providing flexibility for system administrators to include various authorized IPs or IP ranges for seamless access control. Users attempting to access the workspace from IPs outside defined ranges are restricted from entry. Cloud system admins can :ref:`configure IP filtering <manage/cloud-ip-filtering:configure ip filtering>` through their Mattermost System Console.
+
+Encryption
+~~~~~~~~~~~
+
+Mattermost provides encryption-in-transit and encryption-at-rest capabilities. Mattermost supports :doc:`TLS encryption </install/setup-tls>`, including AES-256 with 2048-bit RSA on all data transmissions, between Mattermost client applications and the Mattermost server. You may either set up TLS on the Mattermost Server or :doc:`install a proxy such as NGINX </install/proxy-to-mattermost-transport-encryption>`, and set up TLS on the proxy.
+
+Connections to :doc:`Active Directory/LDAP </onboard/ad-ldap>` can :ref:`optionally be secured with TLS or stunnel <configure/authentication-configuration-settings:ad/ldap port>`.
+
+Connections to calls are secured with a combination of:
+
+- TLS: The existing WebSocket channel is used to secure the signaling path.
+- DTLS v1.2 (mandatory): Used for initial key exchange. Supports ``TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256`` and ``TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA`` algorithms.
+- SRTP (mandatory): Used to encrypt all media packets (i.e. those containing voice or screen share). Supports ``AEAD_AES_128_GCM`` and ``AES128_CM_HMAC_SHA1_80`` algorithms.
+
+Cloud native exports
+~~~~~~~~~~~~~~~~~~~~
+
+Mattermost supports optional :ref:`filestore configuration settings <configure/experimental-configuration-settings:enable dedicated export filestore target>` to direct compliance and bulk export data to a separate S3 bucket from standard files. This separate bucket can be configured to allow for secure access by Mattermost Cloud teams as well as the workspace admins who manage a given installation. The exports can also be accessed by generating unique download links as needed.
+
+The following diagram provides a high-level view of how this functionality works:
+
+.. image:: ../images/mattermost-cloud-native-export-architecture-diagram.png
+  :alt: An architecture diagram showing a high-level view of how Mattermost Cloud Native exports works.
+
+SMTP
+~~~~
+
+Email sent from Mattermost Cloud Dedicated uses SendGrid, and the connection to SendGrid is encrypted.
+
+Audit and observability
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost Cloud Dedicated provides access to :doc:`audit and system logs </manage/logging>` generated by the application. 
+
+Bring your own cloud
```

Comment:
I am not sure we should include BYOC in this round of doc changes.

Reply:
Is there a specific downside we're trying to avoid if we kept it in? Is it too early in the evaluation phase?